homeworks/serhii.ripchanskyi_Ripchanskiy/11-dependency-injection/vechicle/src/app/app.component.html

```diff
@@ -0,0 +1,11 @@
+<p>Vehicle list:</p>
+<div *ngIf="!areVehiclesLoading">Loading...</div>
```

Comment:
You mixed up the variable status :smile: 

Your logic says:
* if vehicles are NOT loading, you will show Loading
* if vehicles are loading, you will show them

So please, fix the logic here :smile: 

**Hint:** it's all about `<div *ngIf="!areVehiclesLoading">Loading...</div>` and the next tag. Just fix `*ngIf` if both elements